bb

```diff
@@ -1389,5 +1409,6 @@ case "$cmd" in
 	pay) bb_pay $@;;
 	grades) bb_grades "$@";;
 	materials) bb_materials "$@";;
+	announcements) bb_announcements "$@";;
```

Comment:
Interestingly, now GitHub gets this preview right, but it still gets it wrong on the full diff.

git still shows the right thing.